CMakeLists.txt

```diff
@@ -149,6 +149,15 @@ macro (sources_hook)
   endif()
   set_source_files_properties(src/opm/input/eclipse/Python/Python.cpp
                               PROPERTIES COMPILE_FLAGS -Wno-shadow)
+  if(QuadMath_FOUND)
+    get_target_property(qm_defs QuadMath::QuadMath INTERFACE_COMPILE_DEFINITIONS)
+    get_target_property(qm_options QuadMath::QuadMath INTERFACE_COMPILE_OPTIONS)
+    set_source_files_properties(src/opm/material/components/CO2.cpp
+                                src/opm/material/densead/Evaluation.cpp
+                                PROPERTIES COMPILE_DEFINITIONS "${qm_defs}"
+                                COMPILE_OPTIONS "${qm_options}")
```

Comment:
Care to share why it is better to put the properties on the source files and not the binary or even library?

Doesn't this mean that the other modules don't see these properties if they use these files (maybe except for you superbulld)?  Isn't that a problem?

Reply:
They are only used to put symbols in the library. These symbols will only be referenced if building with quadmath. But that makes me realize this might break shared linking, i have to check that.